shell/platform/embedder/tests/embedder_test_context.h

```diff
@@ -116,12 +116,6 @@ class EmbedderTestContext {
 
   using NextSceneCallback = std::function<void(sk_sp<SkImage> image)>;
 
-#ifdef SHELL_ENABLE_VULKAN
-  // The TestVulkanContext destructor must be called _after_ the compositor is
-  // freed.
```

Comment:
Hmmm... @bdero is the reason this wasn't pushed down due to the destruction order dependency between this and `compositor_`?

Reply:
If it turns out it’s unsafe to move this, it’d be good if we could have a test that tests the underlying condition we’re concerned about since it _looks_ like all tests pass on this patch.